rust/tools/authorization-logic/raksha_examples/VisionPipelineAuthLogic

```diff
@@ -75,36 +72,32 @@
         isAccessPath(pathX).
 }
 
-// Maybe we want this part of the policy to sit closer to the particles
-// because sometimes relations will refer to path names ?
-// (We can ostensibly do this as long as we can write
-// `claim <RelationName(...)>` in manifests)
-//
-// NOTE: maybe the "will" relations should refer to handles rather than tags, 
-// but the "may"s still refer to the tags, and we figure out which tags to
-// get permissions about based on the taint analysis (for now both still 
-// operate on the tags).
 "ImageDetector" says "ImageDetector"
-    will("do_pure_computation", "raw_video_tag").
+    will("do_pure_computation", "id_sensor_packet.camera_feed").
```

Comment:
Don't you still need to check some properties of the handle?

Reply:
This is just a line that would go in the manifest (or app behavior spec) where the specifier says how the data on this handle is going to be used.

 Eventually, the check described in the comments at the bottom of this file will check the taint analysis on these handles to figure out which "may" facts it needs to check against. (Any "wills" done on handles need to come with a "may" by the owner of the taint tags on the handles, and more complicated proofs involving delegation can be used to satisfy the may part).

Reply:
By the Check, I believe you are referring to this:
```
//          `P says will <RELATION> using <HANDLE>`
//      prove
//          `ownsTag(Q, <TAG>)`
//          `Q says may <RELATION> using <TAG>`
```
How is the `<TAG>` for the `<HANDLE>` in `will` computed? I don't recall any datalog relations that connect the `<HANDLE>` with the `<TAG>`?

Reply:
That's the check I am referring to, yes. The taint analysis in `taint.dl` connects taint tags to paths. We would need to apply the "will" facts to handles, then when the particle is instantiated, substitute the handle for the real path (just like we do/need to do for the claimHasTag facts).

Reply:
(It might be that I'm not properly using the terms handle/paths, but hopefully this is close enough to get the idea across ?)